src/lib/mina_base/party.ml

```diff
@@ -169,10 +169,10 @@ module Body = struct
       Poly.t
 
     let to_input ({ pk; update; token_id; delta } : t) =
-      List.reduce_exn ~f:Random_oracle_input.append
+      List.reduce_exn ~f:Random_oracle_input.Chunked.append
         [ Public_key.Compressed.Checked.to_input pk
         ; Update.Checked.to_input update
-        ; Impl.run_checked (Token_id.Checked.to_input token_id)
+        ; Token_id.Checked.to_input token_id
```

Comment:
Why no `run_checked` ?

Reply:
`Token_id.Checked.to_input` used to add some constraints in order to get the bit representation of the underlying field element. We get to avoid that now, because we can use the field element directly, and so we don't need a `run_checked` to handle adding those constraints.